swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Clothing(Item):
+    '''
+        A Clothing-type Item: the finest clothing you could wear.
+        
+        Attributes:
+            category (str): item type
+            condition (float): item quality (range 0-5; default 0.0)
+    '''
+    def __init__(self, category="Clothing", condition=0.0):
+        super().__init__(category, condition)
```

Comment:
Great use of `super` here! Consider removing the `category` from the constructor arguments on line 10. We know we always want it to be "Clothing" so we don't want to allow changing it when instantiating an instance. We can instead directly pass the string literal "Clothing" to the parent class's constructor.